cleanco/clean.py

```diff
@@ -16,21 +16,37 @@
 from collections import OrderedDict
 import re
 import unicodedata
-from .termdata import terms_by_type, terms_by_country
+from iso20275 import ElfEntry
 
 
 tail_removal_rexp = re.compile(r"[^\.\w]+$", flags=re.UNICODE)
 
 
-def get_terms():
+def get_unique_terms():
     "retrieve all unique terms from termdata definitions"
-    ts = functools.reduce(operator.iconcat, terms_by_type.values(), [])
-    cs = functools.reduce(operator.iconcat, terms_by_country.values(), [])
-    return set(ts + cs)
+    terms = []
+
+    for elf_code, values in Elf.items():
+        entity_codes = Elf[elf_code][0].local_abbreviations
+        if ";" in entity_codes:
```

Comment:
Could splitting & adding items be done more effectively by this one-liner ?
`terms.extend(entity_codes.split(';'))`

Reply:
Changed to be a one-liner, in both clean.py and classify.pt.